.github/workflows/code-review.yaml

```diff
@@ -52,6 +52,7 @@ jobs:
           echo "COSMOSDB_NAME=${{github.run_id}}" >> $GITHUB_ENV
           echo "COSMOSDB_CONNECTION_STRING=${{secrets.COSMOSDB_CONNECTION_STRING}}" >> $GITHUB_ENV
           echo "MONGODB_NAME=${{github.run_id}}" >> $GITHUB_ENV
+          echo "MESSAGEQUEUE_TOPIC=integration-topic" >> $GITHUB_ENV
```

Comment:
Do we need this variable on GITHUB_ENV?

Reply:
yes, is just to use a different topic from the default one